integration-tests/hibernate-reactive-panache/src/test/java/io/quarkus/it/panache/reactive/PanacheFunctionalityTest.java

```diff
@@ -182,8 +199,65 @@ public void testBug7102() {
                 });
     }
 
-    @Transactional
+    @ReactiveTransactional
     Uni<Person> getBug7102(Long id) {
         return Person.findById(id);
     }
+
+    @DisabledOnNativeImage
+    @TestReactiveTransaction
+    @Test
+    @Order(100)
+    public void testTestTransaction(UniAsserter asserter) {
+        asserter.assertNotNull(() -> Panache.currentTransaction());
+        asserter.assertEquals(() -> Person.count(), 0l);
+        asserter.assertNotNull(() -> new Person().persist());
+        asserter.assertEquals(() -> Person.count(), 1l);
+    }
+
+    @DisabledOnNativeImage
```

Comment:
What's wrong with native-image compatibility?

Reply:
In the native test case you don't have access to the running Vert.x instance where you can schedule the execution of the test method